src/Pages/DeckList/DeckList.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from 'react';
+import SearchBar from '../../Shared/SearchBar/SearchBar';
+import SearchPanel from '../../Shared/SearchPanel/SearchPanel';
+import DeckResult from './DeckResult/DeckResult';
+
+class DeckList extends Component {
+    state = { 
+        searchResult: [
+            {id: 1, name: "name1", proto: "very nice prototype", date: "2 april 2019", author: "niceGuy77"},
+            {id: 15,name: "name5", proto: "d", date: "4 september 2019", author: "niceGuy77"},
+            {id: 2,name: "name2", proto: "a", date: "6 april 2019", author: "niceGuy77"},
+            {id: 3,name: "name3", proto: "bbbbbbbbbb", date: "12 may 2019", author: "niceGuy77"},
+            {id: 5,name: "name5", proto: "d", date: "4 september 2019", author: "niceGuy77"},
+            {id: 4,name: "name4", proto: "cc cccc", date: "6 april 2019", author: "niceGuy77"},
+            {id: 6, name: "name1", proto: "very nice prototype", date: "2 april 2019", author: "niceGuy77"},
+            {id: 11, name: "name1", proto: "very nice prototype", date: "2 april 2019", author: "niceGuy77"},
+            {id: 10,name: "name5", proto: "d", date: "4 september 2019", author: "niceGuy77"},
+            {id: 7,name: "name2", proto: "a", date: "6 april 2019", author: "niceGuy77"},
+            {id: 8,name: "name3", proto: "bbbbbbbbbb", date: "12 may 2019", author: "niceGuy77"},
+            {id: 13,name: "name3", proto: "bbbbbbbbbb", date: "12 may 2019", author: "niceGuy77"},
+            {id: 9,name: "name4", proto: "cc cccc", date: "6 april 2019", author: "niceGuy77"},
+            {id: 12,name: "name2", proto: "a", date: "6 april 2019", author: "niceGuy77"},
+            {id: 14,name: "name4", proto: "cc cccc", date: "6 april 2019", author: "niceGuy77"},
+        ]
+     }
+
+    inputChanged(event) {
+        console.log(event.target.value);
```

Comment:
Console logs will still be usefull while connecting to api

Reply:
Ok